### DISCUSSION TITLE
13. Ne pas afficher les documents à l'intérieur des pages.md

### DISCUSSION THREAD

**Discussion by dNicolle** [cnumr/best-practices-wordpress#31] [Bonnes pratiques] [answered]
[ 13. Ne pas afficher les documents à l'intérieur des pages](https://github.com/cnumr/fiches-wordpress-greenit/blob/main/best-practices-WordPress/fiches/13.%20Ne%20pas%20afficher%20les%20documents%20%C3%A0%20l'int%C3%A9rieur%20des%20pages.md)

**Comment by florinesueur** [comment, marked as answer]
Une issue a été créée.
